Assets/HoloToolkit-Examples/Input/Scripts/MicStreamDemo.cs

```diff
@@ -53,7 +53,6 @@ public class MicStreamDemo : MonoBehaviour
         private float minObjectScale = .3f;
 
         private bool isRunning;
-
         public bool IsRunning
```

Comment:
The extra space you took out follows the normal convention and is not improper.

Reply:
So I done this correctly right?

Reply:
So I done this correctly right?